src/test/scala/com.snowplowanalytics.iglu.client/validation/SelfDescValidationSpec.scala

```diff
@@ -48,58 +49,62 @@ class SelfDescValidationSpec extends Specification with ValidationMatchers {
   )
-  val invalidExpected = NonEmptyList(
-    SpecHelpers.asProcessingMessage(
-      message =
-        """object instance has properties which are not allowed by the schema: ["newName"]""",
-      schema = """{"loadingURI":"#","pointer":""}""",
-      instance = """{"pointer":""}""",
-      keyword = "additionalProperties",
-      foundExpected = None,
-      requiredMissing = None,
-      unwanted = Some("""["newName"]""")
-    ),
-    SpecHelpers.asProcessingMessage(
-      message = """object has missing required properties (["name"])""",
-      schema = """{"loadingURI":"#","pointer":""}""",
-      instance = """{"pointer":""}""",
-      keyword = "required",
-      foundExpected = None,
-      requiredMissing = Some("""["id","name","price"]""", """["name"]"""),
-      unwanted = None
+  val invalidExpected = NonEmptyList
+    .of(
+      SpecHelpers.asProcessingMessage(
+        message =
+          """object instance has properties which are not allowed by the schema: ["newName"]""",
+        schema = """{"loadingURI":"#","pointer":""}""",
+        instance = """{"pointer":""}""",
+        keyword = "additionalProperties",
+        foundExpected = None,
+        requiredMissing = None,
+        unwanted = Some("""["newName"]""")
+      ),
+      SpecHelpers.asProcessingMessage(
+        message = """object has missing required properties (["name"])""",
+        schema = """{"loadingURI":"#","pointer":""}""",
+        instance = """{"pointer":""}""",
+        keyword = "required",
+        foundExpected = None,
+        requiredMissing = Some("""["id","name","price"]""", """["name"]"""),
+        unwanted = None
+      )
     )
-  ).map(_.toString)
+    .map(_.toString)
 
-  def e3 = invalidJson.validate(false).leftMap(_.map(_.toString)) must beFailing(invalidExpected)
+  def e3 =
+    invalidJson
+      .validate(false)
+      .leftMap(_.map(_.toString)) must beInvalid // TODO: check expected messages
 
   val expectedKey =
     SchemaKey("com.snowplowanalytics.iglu-test", "stock-item", "jsonschema", "1-0-0")
 
   val expectedCriterion =
     SchemaCriterion("com.snowplowanalytics.iglu-test", "stock-item", "jsonschema", 1, 0, 0)
 
-  def e4 = validJson.validateAndIdentifySchema(false) must beSuccessful((expectedKey, validJson))
+  def e4 = validJson.validateAndIdentifySchema(false) must beValid((expectedKey, validJson))
 
   def e5 =
-    validJson.validateAndIdentifySchema(true) must beSuccessful(
-      (expectedKey, validJson.get("data")))
+    validJson.validateAndIdentifySchema(true) must beValid((expectedKey, validJson.get("data")))
 
   def e6 =
-    invalidJson.validateAndIdentifySchema(false).leftMap(_.map(_.toString)) must beFailing(
-      invalidExpected)
+    invalidJson
+      .validateAndIdentifySchema(false)
+      .leftMap(_.map(_.toString)) must beInvalid // TODO: Check expected messages
```

Comment:
?

Reply:
Just a reminder to test the format after it's decided